UI/Navigation/Common/shared.targets

```diff
@@ -0,0 +1,7 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="CopySharedConfigFiles" BeforeTargets="BeforeBuild">
+    <Message Text="Copying config files to solution" Importance="high" />
+    <Copy SourceFiles="../../Common/global.json" DestinationFolder="../" />
+	<Copy SourceFiles="../../Common/Directory.Build.props" DestinationFolder="../" />
```

Comment:
```suggestion
    <Copy SourceFiles="../../Common/global.json" DestinationFolder="../" />
    <Copy SourceFiles="../../Common/Directory.Build.props" DestinationFolder="../" />
```

Reply:
@nickrandolph Myself and @vatsashah45 thought about using MSBuild. Would this be a good approach to have the packages and sdk versions centralised in one place?